packages/core/src/sfpcommands/package/InstallDataPackageImpl.ts

```diff
@@ -209,11 +208,27 @@ export default class InstallDataPackageImpl {
     
   }
 
-  private buildExecCommand(packageDirectory: string): string {
-    let command = `sfdx sfdmu:run --path ${packageDirectory} -s csvfile -u ${this.targetusername} --noprompt`;
+  private determinePackageType (packageDirectory: string):string {
 
-    SFPLogger.log(`Generated Command ${command}`, null, this.packageLogger);
-    return command;
+    if (fs.pathExistsSync(path.join(packageDirectory, "export.json"))) {
+      SFPLogger.log(
+        `Found export.json in ${packageDirectory}.. Utilizing it as data package and will be deployed using sfdmu`,
+        LoggerLevel.INFO,
+        this.packageLogger
+      );
+      return "sfdmu";
```

Comment:
Perhaps we can use an enumerator for the type of data package

Reply:
True.. will be adding that as part of the next PR